build.gradle

```diff
@@ -601,7 +601,7 @@ task coverage_cpp_report {
 }
 
 task coverage {
-    dependsOn coverage_java_report, coverage_cpp_report
+    dependsOn coverage_java_report
```

Comment:
should we just remove the `coverage` target from `release`? do we actually use coverage data in any non-local-dev workflow (CI, imports, internal pipelines, etc.)?

Reply:
FWIW, the Java coverage report does work and is reported correctly into CodeBuild. Basically, it doesn't have any of the issues related to that cpp coverage target. I don't see a reason to drop it.

Also, removing from release means you have to make changes in at least one other build-script that doesn't use the release target.